ComSemApp/templates/ComSemApp/teacher/edit_released_worksheet.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'ComSemApp/sidebar.html' %}
+
+{% load static %}
+
+{% block content %}
+
+<!-- page head -->
+<form method="POST" id="editWorksheetForm">
+	{% csrf_token %}
+
+	<!-- Page-Title -->
+	<div class="row">
+		<div class="col-sm-12">
+			<div class="page-title-box">
+
+				<h4 class="page-title">
+					{% if not worksheet.id %}New Worksheet{% else %}Edit Worksheet{{ worksheet.number }}{% endif %}
+                    <small>{{ course }}</small>
+				</h4>
+                <a href="{% url 'teacher:course' course.id %}" name='' class='btn btn-sm btn-success float-right release_worksheet_button'>Release Edits</a>
```

Comment:
I feel like this wording might be a little misleading as edits to expressions will already have been 'released' (i.e. saved to the db) whether or not a teacher clicks this link.
How about something like "back to course page" ?

Reply:
I'll make the change and push.